WordPress/Classes/Models/Blog.m

```diff
@@ -505,6 +505,19 @@ - (BOOL)jetpackAccountIsDefaultAccount
     return [defaultAccount isEqual:self.jetpackAccount];
 }
 
+- (nullable NSNumber *)siteID
+{
+    if (self.account)
+    {
```

Comment:
Just a style nit, put the brace on the same line as the if statement (https://github.com/wordpress-mobile/objective-c-style-guide#if-statements)